Gaijinizer patch/Enemies.txt

```diff
@@ -707,11 +706,12 @@ let him be off guard.#MTLed
 感情が高まり、少し興奮気味な口調へ変わる。
 触手の射程はおおよそ五メートルほど。
 > CONTEXT: Enemies/513/BookDescription/4
-Is it because she was remodeled
-When I extend my body to embrace the other person
-Emotions increase, and it changes to a slightly aroused,
-somewhat dignified tone.
-The range of the tentacle is about five meters.#MTLed
+Perhaps because her body was only recently
+converted, when she expands her body to entrap
+her opponent, her emotions well up and her tone
+of voice becomes a little excited.
+Her tentacles have a range of about five metres
+or about 16 feet.#Ron
```

Comment:
Minor flow:
>_**Perhaps it is because her body was only recently
converted, but when she expands her body to entrap
her opponent, her emotions well up and her tone
of voice becomes a little excited.**_